Mathlib/Data/Set/Ncard.lean

```diff
@@ -299,7 +307,7 @@ theorem fiber_ncard_ne_zero_iff_mem_image {y : β} (hs : s.Finite := by toFinite
   convert (ncard_image_of_injective univ Subtype.coe_injective).symm using 1
   · rw [ncard_univ]
   simp
-#align set.nat.card_coe_set_eq Set.Nat.card_coe_set_eq
+#align set.ncardat.card_coe_set_eq Set.Nat.card_coe_set_eq
```

Comment:
Did the mathlib3 declaration name really change? If not, this is wrong. If so, it should probably be part of a forward-port PR.